tensorflow_probability/python/bijectors/real_nvp.py

```diff
@@ -206,24 +209,37 @@ def _bijector_fn(x0, input_depth, **condition_kwargs):
         validate_args=validate_args,
         name=name)
 
+  @property
+  def num_masked(self):
+    if isinstance(self._num_masked, (int, np.integer)):
+      return self._num_masked
+    elif np.issubdtype(type(self._num_masked), np.floating):
+      return int(np.floor(self._input_depth * self._num_masked))
```

Comment:
Should this be floor for both negative and positive values?

Reply:
I think we should round towards zero, so flipping the sign doesn't alter the number of units masked.